examples/Symmetry reduction.jl

```diff
@@ -64,7 +61,7 @@ Certificate.zero_basis(::SymmetricIdeal) = MB.MonomialBasis
 Certificate.get(::SymmetricIdeal, ::Certificate.ReducedPolynomial, poly, domain) = poly
 function Certificate.get(cert::SymmetricIdeal, ::Certificate.GramBasis, poly)
     basis = Certificate.maxdegree_gram_basis(MB.MonomialBasis, MP.variables(poly), MP.maxdegree(poly))
-    R = SymbolicWedderburn.symmetry_adapted_basis(cert.group, basis)
+    R = SymbolicWedderburn.symmetry_adapted_basis(cert.group, basis, permuted)
```

Comment:
did you have a look how the action on monomial basis is handled now in the examples of SymbolicWedderburn?

Reply:
I prefer working with monomials than just exponents so that the user can define the action on the monomial. Working with exponents can be error proned in some case as the order of the variables might not be what the user expect.
In the "Even reduction", it allows the result to be a `Term` which is cleaner than finding a way to combine a coefficient with the vector of exponents.

> There is also no need to define new methods for `ExtensionHomomorphism`.

If we add `permutation` to `SymbolicWedderburn then I'll be able to just add a method for `permutation` that returns a `ScaledPerm` and not add methods for `(::ExtensionHomomorphism)`